niap-cc/Permissions/Tester/app/src/androidTest/AndroidManifest.xml

```diff
@@ -14,46 +14,47 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.android.certifications.niap.permissions.test" >
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-sdk
         android:minSdkVersion="28"
-        android:targetSdkVersion="31" />
+        android:targetSdkVersion="33" />
 
     <instrumentation
         android:name="androidx.test.runner.AndroidJUnitRunner"
         android:functionalTest="false"
         android:handleProfiling="false"
+        android:sharedUserId="android.uid.system"
         android:label="Tests for com.android.certifications.niap.permissions"
         android:targetPackage="com.android.certifications.niap.permissions" />
 
     <uses-permission android:name="android.permission.REORDER_TASKS" />
 
     <application
         android:debuggable="true"
+        android:sharedUserId="android.uid.system"
```

Comment:
Why did this need to be in a sharedUserId with the system?

Reply:
it's not working as you said. removed

Reply:
Thanks! It looks like this is still an attribute under the instrumentation tag in this manifest as well. It looks like that should be ignored by the platform, but just to avoid confusion with anyone else using this, can you remove it from there as well?

Reply:
okay. removed.